app.js

```diff
@@ -4,15 +4,72 @@ const express = require('express');
 const hbs = require('hbs');
 
 // require spotify-web-api-node package here:
+const SpotifyWebApi = require('spotify-web-api-node');
 
-const app = express();
+const spotifyApi = new SpotifyWebApi({
+    clientId: process.env.CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET
+  });
+  
+  // Retrieve an access token
+  spotifyApi
+    .clientCredentialsGrant()
+    .then(data => spotifyApi.setAccessToken(data.body['access_token']))
+    .catch(error => console.log('Something went wrong when retrieving an access token', error));
 
+const app = express();
 app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
 app.use(express.static(__dirname + '/public'));
 
-// setting the spotify-api goes here:
+//setting the spotify-api goes here:
+app.get('/', (rq, rs, next) => {
+    rs.redirect('/home')
+})
 
 // Our routes go here:
+app.get('/home', (rq, rs) => {
+    try {
+        rs.render('homepage')
+    }
+    catch (err) {console.log(err)}
+})
+
+app.get('/artist-search', (rq,rs) => {
+    spotifyApi.searchArtists(rq.query.artist)
+    .then(data => {
+        // console.log('The received data from the API: ', data.body.artists.items[0]);
```

Comment:
also try to remove the console.logs everytime you create a PR its for good practice 🔥  you can keep the ones inside catch